bcolz/carray_ext.pyx

```diff
@@ -734,6 +734,19 @@ cdef class chunks(object):
             scomp = schunk.read(ctbytes)
         return scomp
 
+    def __iter__(self):
+       self._iter_count = 0
+       return self
+
+    def __next__(self):
```

Comment:
Hmm, I am not sure that adding an iterator directly in .chunks would be a good idea.  I think .chunks should be kept mostly hidden (in fact, maybe renaming it to ._chunks) would be good.  Regarding iterating chunks, would:

http://bcolz.blosc.org/reference.html#iterblocks

fill the need?